src/Hedgehog/Property.fs

```diff
@@ -254,9 +224,25 @@ module Property =
         reportBool property
         |> Report.render
 
-    let renderBoolWith (config : PropertyConfig) (p : Property<bool>) : string =
-        reportBoolWith config p
-        |> Report.render
+    /// Set the number of times a property is allowed to shrink before the test
+    /// runner gives up and displays the counterexample.
+    let withShrinks (shrinkLimit : int<shrinks>) (Property (config, gen) : Property<'a>) : Property<'a> =
+        let config = { config with ShrinkLimit = Some shrinkLimit }
+        Property (config, gen)
+
+    /// Restores the default shrinking behavior.
+    let withoutShrinks (Property (config, gen) : Property<'a>) : Property<'a> =
```

Comment:
I think this should be called `withDefaultShrinks`.

Reply:
I'm okay with this, the current name `withoutShrinks` sounds as if no shrinking will occur.

Reply:
I think a better name that `withDefaultShrinks` (which is a syntactic name) is `withUnlimitedShrinking` (which is a semantic name)